config/sync/views.view.facility_services.yml

```
@@ -10625,20 +10731,21 @@ display:
           type_custom_true: ''
           type_custom_false: ''
           not: false
-        edit_node:
-          id: edit_node
-          table: node_field_revision
-          field: edit_node
-          relationship: none
+        nid:
+          id: nid
+          table: node_field_data
+          field: nid
+          relationship: field_regional_health_service
           group_type: group
           admin_label: ''
           entity_type: node
-          plugin_id: entity_link_edit
-          label: ''
+          entity_field: nid
+          plugin_id: field
+          label: URL
           exclude: false
           alter:
             alter_text: true
-            text: '<a href="/node/{{ nid }}/edit" class="button">Edit</a>'
+            text: '/node/{{ nid }}'
```

```suggestion
            text: 'https://prod.cms.va.gov/node/{{ nid }}'
```
Lets give them a fully qualified link to play with.